tutorials/hist/candleplot.C

```diff
@@ -10,7 +11,7 @@
 void candleplot() {
 
    gStyle->SetTimeOffset(0);
-   TRandom *rand = new TRandom();
+   TRandom *randnum = new TRandom();
```

Comment:
Same here - this renaming should /not/ be needed; if it was I'd like to know; I suppose that would be a bug in cling...